README.md

```diff
@@ -71,4 +71,99 @@ Check that you have all the required env vars set in your runtime.
 ### Not ready to serve block N of dataset X
 That error usually means that Subsquid's acrhival node is experiencing some issues.
 If you don't do a huge backfill, set the affected chain gateway like `OPTIMISM_GATEWAY` to an empty string and restart the indexer.
-Additionally, go to Subsquid's discord and open a support ticket like [this one](https://discord.com/channels/857105545135390731/1255998611716575263).
\ No newline at end of file
+Additionally, go to Subsquid's discord and open a support ticket like [this one](https://discord.com/channels/857105545135390731/1255998611716575263).
+
+
+## Adding a New Chain for Indexing
+
+To set up a new chain for indexing contracts and/or transactions, follow these steps:
+
+1. Decide whether you want to track contracts, transactions, or both for the new chain.
+
+2. Choose a unique processor name for your new chain (e.g., 'arbitrum', 'polygon'). This name will be used throughout the setup process. Create a new file in the appropriate directory:
+   - For contracts: `src/chains/contracts/{processorName}.ts`
+   - For transactions: `src/chains/wallets/{processorName}.ts`
+
+3. In this new file, import the necessary functions and handler:
+
+   For contracts:
+   ```typescript
+   import { runProcessor } from "../../utils/ibc-processor";
+   import { handler } from "../../handlers";
+
+   runProcessor('{processorName}', handler)
+   ```
+
+   For transactions:
+   ```typescript
+   import { runProcessor } from '../../utils/ibc-processor'
+   import { handler } from "../../handlers/wallets";
+
+   runProcessor('{processorName}_txs', handler)
+   ```
+
+   Replace '{processorName}' with your chosen unique processor name.
+
+4. Update the configuration file (specified by the `CONFIG_FILE` environment variable) to include the new chain. Add an entry for your chain with the relevant contracts and/or transaction addresses:
+
+   ```yaml
+   {PROCESSOR_NAME}:
+     contracts:
+       - "0x1234567890123456789012345678901234567890"
+     transactions:
+       - "0x0987654321098765432109876543210987654321"
+   ```
+
+5. Set the following environment variables for the new chain:
+
+   - `{PROCESSOR_NAME}_RPC`: The RPC endpoint for the new chain
```

Comment:
eventually maybe this can be a config file?

Reply:
We can add this as a second-level override later if there is a need. For now I prefer to keep it as the env var so it can be reused between different services.